src/common/options/mon.yaml.in

```diff
@@ -918,7 +918,7 @@ options:
 - name: mon_osd_destroyed_out_interval
   type: int
   level: advanced
-  desc: mark any OSD 'out' that has been 'destroy'ed for this long (seconds)
+  desc: mark any OSD 'out' that has been 'destroy'd for this long (seconds)
```

Comment:
This is awkward either way, I think it should be

desc: mark \`\`out\`\` any OSD  that has been in the \`\`destroyed\`\` state for this long (in seconds)